rfcs/text/0017-remove-log-original.md

```diff
@@ -58,9 +64,13 @@ The removal of `log.original` will be considered a breaking change since the fie
 
 - The [`Beats default fields inclusion list`](https://github.com/elastic/ecs/blob/master/scripts/generators/beats_default_fields_allowlist.yml) list should be updated by removing `log.original` if/when Deprecation progresses to Removal
 
-- The logs UI `message` column currently displays `log.original` in the absence of a `message` field. This should be updated to use `event.original` as the substitute field. See [builtin_rules](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.ts) and [associated test](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.test.ts).
+- The logs UI `message` column currently displays `log.original` in the absence of a `message` field. It should be updated to use `event.original` as the substitute field. See [builtin_rules](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.ts) and [associated test](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/server/services/log_entries/message/builtin_rules/generic.test.ts). Kibana team is aware and has a [PR awaiting review](https://github.com/elastic/kibana/pull/102236) which addresses this.
+
+- The breakdown of usage of this field in Kibana source shows:
```

Comment:
Is it possible to fix the indentation here? 
<img width="874" alt="Screen Shot 2021-06-21 at 8 51 45 AM" src="https://user-images.githubusercontent.com/9203126/122764991-0a8a3f80-d26e-11eb-8c29-6b242113b57e.png">

Reply:
Thanks @kgeller, fixed : 
![image](https://user-images.githubusercontent.com/11043991/122768557-011bca00-d2a4-11eb-9831-6b6c06c2e10e.png)